### PR TITLE
Build the middle section

### PR DESCRIPTION
**Pull request by ErikStoupignan**
### Portfolio: mobile version
**Learning objectives**
- Understand how to parse a Figma design to create a UI.
- Use Flexbox to place elements in the page.
- Use CSS Grid to place elements in the page.
- Build a personal portfolio site.

**This branch was used for the design of the following sections:**

- Project.
- About.
- Languages.
- Framework.
- Skills.

I request your authorization to be able to merge the projects, please.

### CODE REVIEW

**Comment by ErikStoupignan** [comment]
Good afternoon, any comment, I am at your disposal.

Have a great day!

File: index.html [hunk 1]
```diff
@@ -48,8 +50,238 @@ <h1 class="main-name">Erik Stoupignan</h1>
         <div class="prueba">
             <img src="/assets/image_geometry_2.png" alt="Decorative photo" class="background-img2">
         </div>
-
     </main>
+
+    <!-- Proyects section -->
```
Comment (danielufeli) [inline index.html]: - [x] Nice working on your `works` section, kindly make the container of your `works section` a `section` tag for better use of HTML semantics

File: index.html [hunk 2]
```diff
@@ -48,8 +50,238 @@ <h1 class="main-name">Erik Stoupignan</h1>
...
+    <h2>Projects</h2>
+    
+    <div class="project">
+
+            <!-- Section  #1-->
+            <div class="container">
+                <img class="box1" src="/assets/Rectangle21.png" alt="Decoration photo">
+                <div class="box box2">
+                    <p class="subtitle">Project names goes here</p>
```
Comment (danielufeli) [inline index.html]: - [x] Kindly make all the project titles use a header tag `hx` for proper use of HTML semantics.

File: index.html [hunk 3]
```diff
@@ -48,8 +50,238 @@ <h1 class="main-name">Erik Stoupignan</h1>
...
+            </div>
+
+            <!-- Section  #2-->
+            <div class="container">
+                <img class="box1" src="/assets/Rectangle21-2.png" alt="Decoration photo">
+                <div class="box box2">
+                    <p class="subtitle">Project names goes here</p>
+                </div>
+                <!-- Button's Code -->
+                <div class="box align box3">
```
Comment (danielufeli) [inline index.html]: - [x] Kindly change the list of the project technologies to a list `ul` and for each of them, a `li`.

File: index.html [hunk 4]
```diff
@@ -48,8 +50,238 @@ <h1 class="main-name">Erik Stoupignan</h1>
...
+                            </p>
+                        </div>
+                    </a>
+                </div>
+            </div>
+
+    </div>
+
+    <!-- About me -->
+    <div class="about">
```
Comment (danielufeli) [inline index.html]: - [x] Nice working on your about section, kindly make the container of the about section a `section` tag for better use of HTML semantics

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @ErikStoupignan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ErikStoupignan) [reply index.html]: @danielufeli Good night Daniel! 
 I have already added the tag section to my "Work section". Thank you for your advice.

And... I have a question about the next point, I sent you a message for slack. Do yo have a chance for a Zoom call?

File: index.html (hunk 2 above)
Comment (ErikStoupignan) [reply index.html]: Ok, I'm already added. Thank you for you advise.

File: index.html (hunk 3 above)
Comment (ErikStoupignan) [reply index.html]: Ok, I fixed. Thank you for your advice.

File: index.html (hunk 4 above)
Comment (ErikStoupignan) [reply index.html]: Ok, perfect. I already fix. Thank you!

**Comment by ErikStoupignan** [comment]
Thank you very much for your comments and observations. I have already attended and corrected all of them.

Have a great day!

File: index.html [hunk 5]
```diff
@@ -48,8 +50,252 @@ <h1 class="main-name">Erik Stoupignan</h1>
...
+                        <!-- Button's Code -->
+                        <div class="box align box3">
+                            <p class="code-button">HTML/CSS</p>
+                        </div>
+                        <div class="box align box4">
+                            <p class="code-button">Ruby on Rails</p>
+                        </div>
+                        <div class="box align box5">
+                            <p class="code-button">JavaScript</p>
+                        </div>
```
Comment (muneebulrehman) [inline index.html]: Hi @ErikStoupignan , what your previous reviewer meant was to wrap this section (technologies used) in a list tag and make each technology a list item, instead, you made the whole work section a list and each card a list item. Kindly fix it.

File: index.html [hunk 6]
```diff
@@ -48,8 +50,252 @@ <h1 class="main-name">Erik Stoupignan</h1>
...
+        <div class="square skill-color">
+            <p class="skill-font-title">Skills</p>
+            <ul class="skill-font-ul">
+                <li>Codekit</li>
+                <li>GitHub</li>
+                <li>GitLAb</li>
+                <li>Terminal</li>
+            </ul>
+        </div>
+    </div>
```
Comment (muneebulrehman) [inline index.html]: This whole section is the about-me section, kindly wrap this whole section in a `section` tag.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @ErikStoupignan , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- No linter errors
- Good PR description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (ErikStoupignan) [reply index.html]: Thankk you @muneebulrehman.
You are absolutely right, sorry. I didn't think things clear.

File: index.html (hunk 5 above)
Comment (ErikStoupignan) [reply index.html]: @muneebulrehman Hello, Thanks for explaining it to me. Honestly, I had not understood but thanks to your comments, I was able to fix it.

**Comment by ErikStoupignan** [comment]
Done, all observations have been taken care of and fixed.
This is my last chance to apply for review, so I try harder to make sure everything is perfect.
Have a nice night and a happy weekend.

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @ErikStoupignan      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ErikStoupignan** [comment]
Excellent! Thank you @KayLemba. I am very happy to read that my project is approved. Have a great weekend!

**Pull request merged**